### DISCUSSION TITLE
Azure Function App PowerShell v7.2

### DISCUSSION THREAD

**Discussion by myatix** [pnp/powershell#2639] [Q&A]
Hi All,

I am implmenting a new Azure Function App and I can no longer choose to downgrade to PowerShell 7.0 to support PnP.PowerShell modules.

![image](https://user-images.githubusercontent.com/48433526/207302844-f6d31d6a-4f34-4728-89e9-2f166088605b.png)

Can anyone tell me if PnP.PowerShell is now compatible with PowerShell 7.2 and if so which version?

Hope someone can shed some light on this! Thanks!

**Comment by Amitkumar028** [comment]
PnP.PowerShell 1.7.0 is currently best with PowerShell 7.2 in AZ functions
